examples/supermassive-todomvc/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "ts-node": {
     "transpileOnly": true,
     "compilerOptions": {
-      "module": "ESNext",
+      "module": "commonjs",
```

Comment:
@freiksenet Needed to make this change, which makes sense to me as this is all about running web pack in Node and Node doesn't properly support es6 modules. Am I missing something?

Reply:
You should pull latest version, it has a fix that works.

Reply:
```  "ts-node": {
    "transpileOnly": true,
    "compilerOptions": {
      "module": "ESNext",
      "target": "ES5",
      "moduleResolution": "Node"
    },
    "moduleType": {
      "webpack.config.ts": "cjs"
    }
  },```